site/pages/join-the-working-group.md

```diff
@@ -0,0 +1,39 @@
+---
+title: FAQ
+description: Join the engineering competencies working group
+permalink: /docs/join-the-working-group
+layout: o-layout-docs
+---
+
+# {{page.title}}
+
+The engineering competencies have been planned, written and rolled out by a working group consisting of the following people:
+
+- [Alice Bartlett](https://people-finder.in.ft.com/search?name=Alice%20Bartlett) - Principal Engineer, Customer Products
+- [Mark Barnes](https://people-finder.in.ft.com/search?name=mark%20barnes) - Technical Officer, Foundation Services
+- [Laura Carvajal](https://people-finder.in.ft.com/search?name=laura%20carvajal) - Principal Engineer, Internal Products
+- [Rowan Manning](https://people-finder.in.ft.com/search?name=Rowan%20Manning) - Principal Engineer, Origami
+- [Rhys Evans](https://people-finder.in.ft.com/search?name=rhys%20evans) - Principal Engineer, Reliability Engineering
+
```

Comment:
- [Tom Kennedy](https://people-finder.in.ft.com/search?name=Tom%20Kennedy) - Project Support Administrator